### ISSUE TITLE
Create k8s cronjob manifest example

### ISSUE THREAD

**Issue by hoeghh** [jonasvinther/medusa#31]
We want an example manifest for a Kubernetes cronjob that uses medusa to backup a vault instance.

Something like : 

```
apiVersion: batch/v1beta1
kind: CronJob
metadata:
  name: medusa
spec:
  schedule: "*/1 * * * *"
  jobTemplate:
    spec:
      template:
        spec:
          containers:
          - name: hello
            image: jonasvinther/medusa:latest
            imagePullPolicy: IfNotPresent
            command:
            - ./medusa
            - -h
          restartPolicy: OnFailure
```
Then from there we will add env's to control what and where.

**Comment by hoeghh** [comment]
Tried it out, and it does run. 

![medusa-cronjobs](https://user-images.githubusercontent.com/11328908/111321209-01cf0f80-8668-11eb-98f7-a613ffc7dbe0.png)

**Comment by hoeghh** [comment]
This seems to work with a configmap and a secret. Its using a hostPath right now to keep it simple. People then needs to add their own volume.

```
---
apiVersion: v1
kind: Secret
metadata:
  name: vault
type: Opaque
stringData:
  token: 00000000-0000-0000-0000-000000000000
---
apiVersion: v1
kind: ConfigMap
metadata:
  name: vault
data:
  vault-url: "https://192.168.86.41:8201"
  vault-path: "secret/A"
  vault_skip_verify: "true"
---
apiVersion: batch/v1beta1
kind: CronJob
metadata:
  name: medusa
spec:
  schedule: "* * * * *"
  jobTemplate:
    spec:
      template:
        spec:
          securityContext:
            runAsUser: 1000
          containers:
          - name: medusa
            image: jonasvinther/medusa:latest
            imagePullPolicy: IfNotPresent
            command: ["./medusa", "export", "$(VAULT_PATH)", "-o", "/backup/backup.vault"]
            env:
            - name: VAULT_SKIP_VERIFY
              valueFrom:
                configMapKeyRef:
                  name: vault
                  key: vault_skip_verify
            - name: VAULT_ADDR
              valueFrom:
                configMapKeyRef:
                  name: vault
                  key: vault-url
            - name: VAULT_PATH
              valueFrom:
                configMapKeyRef:
                  name: vault
                  key: vault-path
            - name: VAULT_TOKEN
              valueFrom:
                secretKeyRef:
                  name: vault
                  key: token
            volumeMounts:
              - name: backup
                mountPath: /backup
          restartPolicy: OnFailure
          volumes:
          - name: backup
            hostPath:
              path: /backup
              type: DirectoryOrCreate

```

**Comment by hoeghh** [comment]
Fixed with #34
